tests/run_cookstyle.rb

```diff
@@ -12,9 +12,9 @@
     if @files.empty?
       puts 'Linting not performed. No ruby files changed.'
     else
-      puts "Running rubocop for changed files: #{@files}"
+      puts "Running cookstyle for changed files: #{@files}"
       result = system \
-        "bundle exec rubocop --config .rubocop.yml #{@files}"
+        "bundle exec cookstyle --config .rubocop.yml #{@files}"
```

Comment:
Do we want to add a `--fail-level SEVERITY` here to fail out on certain offenses?

Reply:
If you want this stuff to all fail you can set the fail level to R for refactor

Reply:
Understood, just wondering if that's what we should do here to ensure folks make the changes before merging. Rather than just allowing the PR to go through.